src/reactor.cc

```diff
@@ -357,7 +357,7 @@ void Reactor::Tock() {
     return;
   }
 
-  if (cycle_step >= cycle_time + refuel_time && core.count() == n_assem_core) {
+  if (cycle_step >= cycle_time + refuel_time && core.count() == n_assem_core && discharged == true) {
```

Comment:
could you add a small comment about this if statement, to make sure people in the future understand all the condition in it ?

Reply:
sure!

If you approve of the comment, everything should be settled right? Do I have to do something then, eg., close the pull request? I am new to Github so I'm not sure about what to do.